projects/swimlane/ngx-ui/src/lib/components/json-editor/json-editor-flat/json-editor-node-flat/json-editor-node-flat.component.html

```diff
@@ -0,0 +1,125 @@
+<div *ngIf="model !== undefined" class="json-tree-node-flat">
+  <div class="node">
+    <div class="indention" *ngIf="level > 0" [style.left]="-1 * (level* 20) + 'px'">
+      <span class="separator" *ngFor="let separator of indentationArray"></span>
+    </div>
+
+    <div class="error-box" *ngIf="!valid"></div>
+
+    <span class="required-indicator" *ngIf="required" [class.invalid]="!childrenValid">*</span>
+
+    <div class="node-content">
+      <div class="node-info">
+        <div class="info-name" *ngIf="schema?.propertyName">
+          <span class="name">{{ schema?.propertyName | decamelize }}</span>
+        </div>
+
+        <div class="info-type">
+          <span class="type">{{ schema?.type | titlecase }}</span>
+          <ngx-icon *ngIf="schema?.propertyName" class="dot-separator" fontIcon="circle-filled"></ngx-icon>
+          <span class="property-name">{{ schema?.propertyName | cammeltosnake }}</span>
+        </div>
+
+        <span class="description">{{ schema?.description }}</span>
+      </div>
+
+      <div class="node-input">
+        <!-- Number | Integer -->
+        <div *ngIf="schema?.type === 'number' || schema?.type === 'integer'">
+          <ngx-input type="number" [ngModel]="model" (ngModelChange)="updateModel(parseInt($event))"
```

Comment:
Can probably do better coercion, since this will convert floats to integers too, but that's a bigger undertaking for the whole library, so extracted it into a ticket: https://github.com/swimlane/ngx-ui/issues/321

Maybe change this to `Number($event')` for now? Don't know if that even works in templates.